lib/i18n.js

```diff
@@ -79,7 +79,15 @@ function Localize () {
                 return localized;
             }
         }
-        return dict || defaultLang || key;
+        var value = dict || defaultLang || key;
+        if (!!value) {
+            var macroReplace = function (pattern, term) {
+                var replacement = !!macros ? macros[term] : false;
+                return replacement ? replacement : pattern;
+            };
+            value = value.replace(/{{\s*([^}\s]+)\s*}}/g, macroReplace);
```

Comment:
You sure you want to leave out whitespace from the group? e.g. `app name` won't work, but `appname` or `app.name` will.